### PR TITLE
[Docs] Update Pocket TestNet nodes

### PR DESCRIPTION
**Pull request by Olshansk**
1. Remove Nodefleet as the TestNet maintainer
2. Add cryptonodes & stakenodes as TestNet Maintainer

### CODE REVIEW

**Review by breezytm — COMMENTED** [review]
The following sections need to be removed 
```Validator List w/ Pocket RPC```
```Validators w/ Tendermint RPC```

The following sections need to be updated to remove the extra `/` in the examples after the load balancer URLs
```curl https://tendermint.testnet.pokt.network/lb/6d6f727365//net_info```
```curl https://tendermint.testnet.pokt.network/lb/6d6f727365//status```

The `TestNet Seeds` list needs to be updated to reflect the following 
```b3d86cd8ab4aa0cb9861cb795d8d154e685a94cf@seed1.testnet.pokt.network:26663
5b0107a5252f6a037eed7f5c24a7d916e4dd93bd@testnet_seed_4.cryptonode.tools:16646```

**Comment by Olshansk** [comment]
@breezytm Updated the docs. 

Thanks @breezytm

File: doc/guides/testnet.md [hunk 1]
```diff
@@ -2,178 +2,63 @@
...
-https://node2.testnet.pokt.network/
-https://node3.testnet.pokt.network/
-https://node4.testnet.pokt.network/
-https://node5.testnet.pokt.network/
-https://node6.testnet.pokt.network/
-```
+### BreezyTm (StakeNodes)
 
-### Example Queries
+`breezytm | Stakenodes (277262895459336194)` has made his Validator's RPC + Tendermint
```
Comment (Olshansk) [inline doc/guides/testnet.md]:
```suggestion
`breezytm | Stakenodes (277262895459336194)` Validator RPC + Tendermint
```

File: doc/guides/testnet.md [hunk 2]
```diff
@@ -2,178 +2,63 @@
...
+- **RPC**: [rpc.testnet.pokt.network/lb/6d6f727365](https://rpc.testnet.pokt.network/lb/6d6f727365)
+- **Tendermint**: [rpc.testnet.pokt.network/lb/6d6f727365](https://tendermint.testnet.pokt.network/lb/6d6f727365)
 
-```bash
-curl -X POST https://node1.testnet.pokt.network/v1/query/state | tee query_state.json | jq
-```
+### Ian (Cryptonode.tools)
 
-Output:
+`Ian | cryptonode.tools (693644362575511573)` also made his Validator's endpoints available here
```
Comment (Olshansk) [inline doc/guides/testnet.md]:
```suggestion
`Ian | cryptonode.tools (693644362575511573)` Validator endpoints available here:
```

**Pull request merged**